src/Service/QuerySearchService.php

```diff
@@ -81,6 +81,16 @@ public function isSortable(): bool
         return true;
     }
 
+    /**
+     * @param string[] $ids
+     *
+     * @return QuerySearch[]
+     */
+    public function getByIds(array $ids): array
+    {
+        return $this->querySearchRepository->getByIds($ids);
+    }
+
```

Comment:
I had in mind to use the QuerySearch's name. It's more human readable than a uid.

Reply:
OK, my bad. I didn't see any unicity logic for the QuerySearch name and mainly, I thought it would be use only via the QuerySearch picker.